prettier.config.js

```diff
@@ -0,0 +1,9 @@
+/** @type {import('prettier').Config} */
+module.exports = {
+  trailingComma: 'es5',
+  bracketSpacing: true,
+  tabWidth: 2,
+  semi: false,
+  singleQuote: true,
+  arrowParens: 'always',
+};
```

Comment:
__*Preference*__
6/10

I know you took this from our internal config. I might lean to just using the default config for prettier here instead. There are a couple things I may prefer in this (and a couple I don't), but I think it might be less disruptive here

Reply:
Yes, I was thinking about this, too. Fortunately, it's very easy to change, and it's not applied in this PR.

We can leave this decision for the formatting draft PR that comes after this one.